adapters/rhythmone/rhythmonetest/exemplary/simple-video.json

```diff
@@ -4,61 +4,92 @@
     "imp": [
       {
         "id": "test-imp-id",
-	"video": {
-		"mimes": ["video/mp4"],
-		"minduration": 1,
-		"maxduration": 2,
-		"protocols": [1, 2, 5],
-		"w": 1020,
-		"h": 780,
-		"startdelay": 1,
-		"placement": 1,
-		"playbackmethod": [2],
-		"delivery": [1],
-		"api": [1, 2, 3, 4]
-	   },
-	"ext": {
-		"bidder": {
-			"placementId": "72721",
-			"path": "mvo",
-			"zone": "1r"
-	 	}
-	  }  
-       }
+        "video": {
+          "mimes": [
+            "video/mp4"
+          ],
+          "minduration": 1,
+          "maxduration": 2,
+          "protocols": [
+            1,
+            2,
+            5
+          ],
```

Comment:
Yeah, I agree with you. Small lists like this can be on their own line. This for me happened automatically via a JSON formatter. I tried to play around with it's settings to see if I can configure it to not do that but didn't have much luck. Changed it manually for this file but there could be others that still have such lists auto-formatted.

If you know of a VSCode JSON formatter where I can control this, please lemme know. I will be happy to have that in my setup and will also update all the other files.

Reply:
Which formatter are you using?

Reply:
I use Beautify.